src/vm/fieldmarshaler.h

```diff
@@ -1982,6 +1989,132 @@ class FieldMarshaler_DateTimeOffset : public FieldMarshaler
 
 #endif // FEATURE_COMINTEROP
 
+class FieldMarshaler_FixedBuffer : public FieldMarshaler
+{
+public:
+
+    FieldMarshaler_FixedBuffer(ULONG bufferSize, MethodTable* pMT)
+    {
+        size = bufferSize;
```

Comment:
Please us initializer list for so `size` can be made `const`. (i.e. `size{ bufferSize }`)